IopHomeNode.proto3

```diff
@@ -524,15 +524,16 @@ message ListHomeNodePlansResponse {
 
 // Description of a home node hosting plan.
 message HomeNodePlan {
-  bytes planId = 1;                                // GUID of the plan. If a node wants to change the plan, it has to change the GUID as well,
-                                                   // thus effectively creating a new plan.
-  string identityType = 2;                         // Type of identity this plan is created for. This can include "*" as a wildcard character
+  bytes planId = 1;                                // Identifier of the plan is an SHA1 hash of the HomeNodePlan structure in Google Protobuf v3 encoding 
+                                                   // with planId initialized to a byte array of 20 zero bytes.
```

Comment:
I know that we suggested changing this to an sha1 hash, thus I perform self-critisism here. After some prototyping and more consideration, I realized that this would break separation of concerns and would tie business logic (application) and protobuf (presentation) layers together. So probably it's not as good as it seemed. Anyway, It would make sense only if just the planId, not the whole HomeNodePlan object is specified in HomeNodeRequestRequest, but in the current state does not seem very useful.